### PR TITLE
adds dropdown as a settings format

### LINKED ISSUE #437: Update Settings to include Bani Length for SG
When we surface SG there needs to be a way to choose Bani Length

### PR DESCRIPTION
**Pull request by inderpreetsingh**
Fixes #437 , adds new dropdown format to settings.

### CODE REVIEW

File: www/js/settings.json [hunk 1]
```diff
@@ -74,16 +74,24 @@
...
+      }
+    }
+  },
+  "toolbar": {
+    "title": "Toolbar",
+    "settings": {
       "sunder-gutka": {
-        "title": "Sundar Gutka",
-        "type": "range",
+        "title": "Sunder Gutka Settings",
```
Comment (maneetpaul) [inline www/js/settings.json]: This should be “Gurbani Settings” since vishraams will be in the same section

File: www/js/settings.json [hunk 2]
```diff
@@ -74,16 +74,24 @@
...
         "options": {
           "bani-length": {
             "title": "Bani Length",
-            "max": 4,
-            "min": 1,
-            "step": 1
+            "options": {
+              "small": "Small",
+              "medium": "Medium",
+              "large": "Large",
```
Comment (maneetpaul) [inline www/js/settings.json]: This should be Long and then Very Long to stay consistent with Sundar Gutka mobile app

Comment (inderpreetsingh) [reply www/js/settings.json]: done!

File: www/js/settings.json [hunk 3]
```diff
@@ -74,16 +74,24 @@
...
           }
         }
-      },
-      "sunder-gutka": {
-        "title": "Sundar Gutka",
-        "type": "range",
+      }
+    }
+  },
+  "toolbar": {
```
Comment (tsingh777) [inline www/js/settings.json]: would it be better if this was closer to fields to display/display options? might not matter.

**Pull request merged**